src/main/scala/com/horizon/exchangeapi/tables/Services.scala

```diff
@@ -97,6 +99,7 @@ object ServicesTQ {
   def getMatchHardware(service: String): Query[Rep[String], String, Seq] = rows.filter(_.service === service).map(_.matchHardware)
   def getRequiredServices(service: String): Query[Rep[String], String, Seq] = rows.filter(_.service === service).map(_.requiredServices)
   def getUserInput(service: String): Query[Rep[String], String, Seq] = rows.filter(_.service === service).map(_.userInput)
+  def getUserSecrets(service: String): Query[Rep[String], String, Seq] = rows.filter(_.service === service).map(_.secrets)
```

Comment:
Another minor thing, but to follow the naming conventions this should be named `getSecrets`